docs/msf/managing-store-configuration.mdx

```diff
@@ -127,7 +161,7 @@ Accept: application/json
 
 To update channel-specific settings, send a request to the same endpoint along with a value for the `channel_id` query parameter.
 
-```http title="Example request: Update store profile settings for channel 123"
+```http filename="Example request: Update store profile settings for channel 123"
```

Comment:
```suggestion
```http filename="Example request: Update store profile settings for channel 123" copy
```